data/categories.txt

```diff
@@ -1249,7 +1249,7 @@ amenity-place_of_worship
 en:place of worship|4temple|sights|U+1F64F|U+26EA
 ru:3храм|церковь|достопримечательность
 uk:3храм|церква|собор|пам’ятка|алтарь|мавзолей|пам’ятні місця
-de:Anbetungsstätte|Sehenswürdigkeit
+de:Anbetungsstätte|Kirche|Dom|Tempel|Kathedrale|Kapelle|Moschee|Sehenswürdigkeit
```

Comment:
I’ve just cleaned them up - they had many old translations not related to churches.
> On Feb 1, 2016, at 19:03, malenki <notifications@github.com> wrote:
> 
> In data/categories.txt <https://github.com/mapsme/omim/pull/1735#discussion_r51436702>:
> 
> > @@ -1249,7 +1249,7 @@ amenity-place_of_worship
> >  en:place of worship|4temple|sights|U+1F64F|U+26EA
> >  ru:3храм|церковь|достопримечательность
> >  uk:3храм|церква|собор|пам’ятка|алтарь|мавзолей|пам’ятні місця
> > -de:Anbetungsstätte|Sehenswürdigkeit
> > +de:Anbetungsstätte|Kirche|Dom|Tempel|Kathedrale|Kapelle|Moschee|Sehenswürdigkeit
> Ok, but Chinese and Japanese seem to be quite verbose, too. (I don't speak those languages, though.)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/1735/files#r51436702>.
>